### PR TITLE
chore: Update Backoffice to backport version v6.6.2

### PR DESCRIPTION
**Pull request by hectoras**
**Associated Jira issue:** [AUT-1927](https://oat-sa.atlassian.net/browse/AUT-1927).

This pull request updates `composer.json` for the February release to point to [v6.6.2](https://github.com/oat-sa/extension-tao-backoffice/releases/tag/v6.6.2) fix version of TAO Backoffice.

### CODE REVIEW

**Review by gabrielfs7 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
         "oat-sa/extension-tao-revision": "10.2.0",
         "oat-sa/extension-tao-mediamanager": "12.21.2",
         "oat-sa/extension-pcisample": "3.4.0",
-        "oat-sa/extension-tao-backoffice": "6.6.1",
+        "oat-sa/extension-tao-backoffice": "6.6.2",
```
Comment (gabrielfs7) [inline composer.json:55]: We are missing composer.lock

Comment (hectoras) [reply composer.json:55]: :facepalm: (on it)

**Review by shpran — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (hectoras) [reply composer.json:55]: :point_right: 3e98ef02023efa204ffa562159379fa82d4e712e

**Comment by hectoras** [comment]
Released as [2022.02.3](https://github.com/oat-sa/tao-community/releases/tag/2022.02.3).

**Pull request merged**